esp-link/config.c

```diff
@@ -35,6 +35,8 @@ FlashConfig flashDefault = {
   .data_bits	= EIGHT_BITS,
   .parity	= NONE_BITS,
   .stop_bits	= ONE_STOP_BIT,
+  .telnet_port1 = 23,
+  .telnet_port2 = 2323,
```

Comment:
@tve So I looked into this a bit - as the code stands now, if I erase the flash... Than flash my current build, the default ports of 23 & 2323 are set(as they are set in this file). How is it accomplishing this?

As far as I can tell, the configRestore() function is where checks for "0" values are checked. 
Why not simply use a loop to check all the flashConfig variables for "0" or NULL and fill them with their flashDefault values if true? (Just trying to learn here - I'm sure you or someone else has a good reason here)